pvlib/transformer.py

```diff
@@ -0,0 +1,115 @@
+"""
+This module contains functions for transformer modeling.
+
+Transformer models calculate AC power output at a different voltage from the
+voltage at the AC input terminals.
+"""
+
+import numpy as np
+
+
+def simple_efficiency(
+        input_power, no_load_loss, load_loss, transformer_rating
+):
+    r'''
+    Calculate the energy at the output terminal of the transformer
+    after taking into account efficiency using a simple calculation.
+
+    The equation used in this function can be derived from [1]_.
+
+    First, assume that the load loss is proportional to the square of output
+    power.
+
+        .. math::
+        L_{load}(P_{out}) = L_{load}(P_{out}) * P^2_{out}
```

Comment:
Blank line here, and indent the equations three spaces to line up under the "m" in "math::"

Reply:
updated spacing and indentation